apps/frontend/src/pages/newEvent.tsx

```diff
@@ -0,0 +1,66 @@
+import { styles } from '@/components/newEventStyles';
+
+export default function newEvent() {
+  return (
+    <div>
+      <div>
+        <div className='items-center rounded-2xl border-4 border-dark-green ml-2.5 mt-4 w-80 text-2xl text-center bg-light-green p-2'>
+          Event létrehozása
+        </div>
+        <div style={styles.AlignStyle}>
+          <div style={styles.StickerStyle}>
+            Név:
+            <div className='w-3/4'>
+              <input id='name' placeholder='Gólyakocsma...' style={styles.InputStyle} />
+            </div>
+          </div>
```

Comment:
Itt sok helyen ugyanaz a kód van, csak más az innerHTML
ezeket is nyugodtan ki lehetne szervezni külön komponensbe